spec/swagger_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.configure do |config|
+  config.swagger_root = Rails.root.join("swagger").to_s
+  config.swagger_docs = {
+    "v1/api_spec.json" => {
+      swagger: "2.0",
+      info: {
+        title: "Provider Events API",
```

Comment:
```suggestion
        title: "API documentation",
```
Sounds like this might end up being more than just the events interface